charms/kfp-persistence/src/charm.py

```diff
@@ -115,6 +115,7 @@ def __init__(self, *args, **kwargs):
                     "NAMESPACE": "",
                     "TTL_SECONDS_AFTER_WORKFLOW_FINISH": "86400",
                     "NUM_WORKERS": "2",
+                    "EXECUTIONTYPE": "Workflow",
```

Comment:
There must be something wrong with the charm that gets published in this PR because I was getting this error:

```
2024-07-04T23:42:12.652Z [pebble] Service "persistenceagent" starting: persistence_agent  --logtostderr=true  --namespace=  --ttlSecondsAfterWorkflowFinish=86400  --numWorker=2  --mlPipelineAPIServerName=kfp-api.kubeflow  --executionType Workflow  --logLevel=info
2024-07-04T23:42:12.679Z [persistenceagent] flag provided but not defined: -executionType
```

but then I explicitly told juju to use the `charmedkubeflow/persistenceagent:2.2.0-8af6d3c` resource `juju refresh kfp-persistence --resource oci-image=charmedkubeflow/persistenceagent:2.2.0-8af6d3c` and it started working.

Reply:
It looks like https://github.com/canonical/bundle-kubeflow/issues/962 is also affecting this. I see the resource that gets attached to this revison of the charm was created on 2024-06-20, so way before the last run of this PR's CI.

Reply:
Actually, I did a second pass on the resource revisions and the dates and the charms are indeed using the right resource revision (the ones created exactly on jun 20, when this PR was created), but I'm not sure then why for schedwf and persistence their images were not correct.